src/directives/AcceptGrant.tsx

```diff
@@ -61,6 +65,20 @@ function AcceptGrant({
                 async () => {
                     trackEvent(`${directiveName}:Complete`, directive);
 
+                    if (mutate_userInfoSummary) {
+                        mutate_userInfoSummary()
+                            .then(() => {
+                                trackEvent(
+                                    `${directiveName}:UserInfoSummary:Mutate:Success`
+                                );
+                            })
+                            .catch(() => {
+                                trackEvent(
+                                    `${directiveName}:UserInfoSummary:Mutate:Error`
+                                );
+                            });
+                    }
```

Comment:
This is kinda weird and one of those places where I am wondering if we should just throw up a FullPageError... cause if we cannot get this call to work we are in big trouble.